api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientImporter.java

```diff
@@ -0,0 +1,69 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.module.registrationcore.api.impl.IdentifierBuilder;
+import org.openmrs.module.registrationcore.api.impl.RegistrationCoreProperties;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiAuthenticator;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiPatientImporter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+public class OpenEmpiPatientImporter implements MpiPatientImporter {
+
+    private final Log log = LogFactory.getLog(this.getClass());
+
+    @Autowired
+    @Qualifier("registrationcore.restQueryExecutor")
+    private RestQueryExecutor queryExecutor;
+
+    @Autowired
+    @Qualifier("registrationcore.identifierBuilder")
+    private IdentifierBuilder identifierBuilder;
+
+    @Autowired
+    @Qualifier("registrationcore.patientBuilder")
+    private PatientBuilder patientBuilder;
+
+    @Autowired
+    @Qualifier("registrationcore.mpiAuthenticator")
+    private MpiAuthenticator authenticator;
+
+    @Autowired
+    @Qualifier("registrationcore.coreProperties")
+    private RegistrationCoreProperties coreProperties;
+
+    @Override
+    public Patient importMpiPatient(String patientId) {
+        OpenEmpiPatientQuery mpiPatient = queryExecutor.getPatientById(authenticator.getToken(), patientId);
+
+        return createPatient(mpiPatient);
+    }
+
+    private Patient createPatient(OpenEmpiPatientQuery mpiPatient) {
```

Comment:
I would call this buildPatient(). When I hear createPatient I presume it is already saved to the DB